translations_to_weblate.sh

```diff
@@ -1,23 +1,15 @@
 #!/bin/bash
-target='*.tsl'
-output='BE_weblate.json'
+MY_PATH="`dirname \"$0\"`"
+TARGET='*.tsl'
+OUTPUT='BE_weblate.json'
 if [[ ! -z $1 ]] ; then
```

Comment:
...but I'd feel better if references to user-provided values were doublequoted, to avoid surprises with empty tokens, embedded whitespace and such...